eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Plutus/Evaluate.hs

```diff
@@ -75,43 +57,35 @@ import NoThunks.Class (NoThunks)
 -- From the specification, Figure 8 "Scripts and their Arguments"
 -- ===============================================================
 
--- | Only the Spending ScriptPurpose contains TxIn
-getSpendingTxIn :: ScriptPurpose era -> Maybe (TxIn (EraCrypto era))
-getSpendingTxIn = getAlonzoSpendingTxIn
-{-# DEPRECATED getSpendingTxIn "In favor of `getAlonzoSpendingTxIn`" #-}
-
--- | Get the Data associated with a ScriptPurpose. Only the Spending
---   ScriptPurpose contains Data. Nothing is returned for the other kinds.
-getDatumAlonzo ::
-  (AlonzoEraTxWits era, AlonzoEraTxOut era, EraTx era) =>
-  Tx era ->
-  UTxO era ->
-  ScriptPurpose era ->
-  Maybe (Data era)
-getDatumAlonzo tx utxo = getAlonzoSpendingDatum utxo tx
-{-# DEPRECATED getDatumAlonzo "In favor of `getAlonzoSpendingDatum`" #-}
-
 -- ========================================================================
 
 -- | When collecting inputs for two phase scripts, 3 things can go wrong.
 data CollectError era
   = NoRedeemer !(ScriptPurpose era)
   | NoWitness !(ScriptHash (EraCrypto era))
   | NoCostModel !Language
-  | BadTranslation !(TranslationError (EraCrypto era))
+  | BadTranslation !(ContextError era)
```

Comment:
I see none of the changes (including this, exporting lookupPlutusScript, removing collectTwoPhaseScriptInputs ) are mentioned in the CHANGELOG, but perhaps they are not intended to be used by anyone else.
`CollectError` I see is used the Utxos rule predicate failure, so maybe it would help to mention this change?

Reply:
They absolutely have to be in the CHANGELOG. Thank you for reminding me to finish adjusting changelogs. I stopped half way through the PR.

Reply:
I've added a bunch of missing changelog entries for `cardano-ledger-alonzo`